src/components/ChallengePane/ChallengePane.js

```diff
@@ -105,6 +106,14 @@ export class ChallengePane extends Component {
 
   resetSelectedClusters = () => this.setState({selectedClusters: []})
 
+  componentDidMount() {
+    this.props.clearSearch()
+    this.props.clearSearchFilters()
+    this.props.setSearchSort({sortBy: 'default'})
+    this.props.setSearchFilters({location: ChallengeLocation.intersectingMapBounds})
+    this.props.clearMapBounds()
+  }
```

Comment:
how is this related to metrics v2?  if it's not and we're just fixing a defect, we should create a new PR